tests/integration/tests/bundles/jammy.yaml

```diff
@@ -0,0 +1,28 @@
+series: jammy
+
+applications:
+  ubuntu:
+    charm: ch:ubuntu
+    num_units: 1
+
+  juju-local:
+    charm: bootstack-charmers-juju-local
+    num_units: 0
+
+  prometheus-juju-exporter:
+    charm: ../../../../prometheus-juju-exporter.charm
+    num_units: 0
+    resources:
+      exporter-snap: ../../../../exporter.snap
```

Comment:
```suggestion
  prometheus-juju-exporter:
    charm: {{ CHARM_LOCATION }}//prometheus-juju-exporter.charm
    num_units: 0
    resources:
      exporter-snap: {{ CHARM_LOCATION }}/exporter.snap
```

Reply:
Not sure if this will work or you need to use local overlays, as I mention in my original comment.